latency/LatencyScanData.py

```diff
@@ -0,0 +1,86 @@
+#!/bin/env python2.7
+
+# -*- coding: utf-8 -*-
+"""
+
+@author: Anastasia and Cameron Bravo (c.bravo@cern.ch)
+
+"""
+import ROOT as r
+import numpy as np
+from array import array
+from optparse import OptionParser
+
+#Define and parse options
+parser = OptionParser()
+
+parser.add_option("-i", "--infilename", type="string", dest="filename", default="LatencyScanData.root",
+                  help="Specify Input Filename", metavar="filename")
+parser.add_option("-o", "--outfilename", type="string", dest="outfilename", default="SCurveFitData.root",
```

Comment:
and if you're making the previous change, then the default here should be changed too